apps/dashboard/src/components/template-store/workflow-sidebar.tsx

```diff
@@ -0,0 +1,193 @@
+import { CreateWorkflowButton } from '@/components/create-workflow-button';
+import { Calendar, Code2, ExternalLink, FileCode2, FileText, KeyRound, LayoutGrid, Users } from 'lucide-react';
+import { motion } from 'motion/react';
+import { ReactNode } from 'react';
+import { Badge } from '../primitives/badge';
+import { WorkflowMode } from './types';
+
+interface WorkflowSidebarProps {
+  selectedCategory: string;
+  onCategorySelect: (category: string) => void;
+  mode: WorkflowMode;
+}
+
+interface SidebarButtonProps {
+  icon: ReactNode;
+  label: string;
+  onClick?: () => void;
+  isActive?: boolean;
+  bgColor?: string;
+  asChild?: boolean;
+  hasExternalLink?: boolean;
+  beta?: boolean;
```

Comment:
not used

Reply:
This will be used in a subsequent suggestion PR